tests/DiaryEntryModelTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+require_once __DIR__ . '/_CustomTestRunnerHooks.php';
+use PHPUnit\Framework\TestCase;
+
+use WTSA1\Engines\Database;
+use WTSA1\Models\DiaryEntry;
+
+class DiaryEntryModelTest extends TestCase {
+
+    public function setUp(): void {
+        \WTSA1\Engines\Database::getInstance()->query("
```

Comment:
I think we can change all uses of ` \WTSA1\Engines\Database` to just `Datebase` since it is `use`d explicity.

Reply:
I'll do it

Reply:
Done at commit https://github.com/IBZTI2018/TI3-WT-SA/commit/7eb60e7cc47d8e26fbea6c371c1fc3bc76a6bfe6